router/boot.go

```diff
@@ -75,9 +84,10 @@ func main() {
 	// FIXME: have to launch cron first so generate-certs will generate the files nginx requires
 	go launchCron()
 
-	waitForInitialConfd(host+":"+etcdPort, timeout)
+	confdHost := strings.Split(etcdPeers[0], "://")[1]
```

Comment:
@laurrentt what exactly are you setting `ETCDCTL_PEERS` to?  That env variable (normally used by etcd itself and re-used here is (or should be) a comma-delimited list of etcd hosts, but should also include the protocol (`http`) and port (4001).  I think it shouldn't matter whether you use IP addresses or DNS name/hostname, but unless `ETCD_PEERS` conforms to the format I described, the line you're referencing won't properly pluck an IP/name and port out of that env variable.